### PR TITLE
oxipng: add page

### PR DESCRIPTION
**Pull request by Anjellyrika**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known): 9.1.2**

[Oxipng](https://github.com/shssoichiro/oxipng) is a multithreaded lossless PNG/APNG compression optimizer written in Rust. This PR adds a page for commonly-used Oxipng commands. 🌟

### CODE REVIEW

**Review by einverne — APPROVED** [review]
Looks good, thanks for the new page.

File: pages/common/oxipng.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`oxipng --out {{path/to/newfile.png}} {{path/to/file.png}}`
+
+- Compress all PNG files in the current directory using multiple threads:
+
+`oxipng "{{*.png}}"`
+
+- Compress a file with a set optimization level (0-6 or max; default is 2):
+
+`oxipng --opt {{level}} {{path/to/file.png}}`
```
Comment (spageektti) [inline pages/common/oxipng.md:20]:
```suggestion
`oxipng --opt {{0|1|2|3|4|5|6|max}} {{path/to/file.png}}`
```

File: pages/common/oxipng.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Compress a PNG file and save the output to a new file:
+
+`oxipng --out {{path/to/newfile.png}} {{path/to/file.png}}`
+
+- Compress all PNG files in the current directory using multiple threads:
+
+`oxipng "{{*.png}}"`
+
+- Compress a file with a set optimization level (0-6 or max; default is 2):
```
Comment (spageektti) [inline pages/common/oxipng.md:18]:
```suggestion
- Compress a file with a set optimization level (default is 2):
```

File: pages/common/oxipng.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`oxipng "*.png"`
+
+- Compress a file with a set optimization level (default is 2):
+
+`oxipng --opt {{0|1|2|3|4|5|6|max}} {{path/to/file.png}}`
+
+- Set the PNG interlacing type (0 removes interlacing, 1 applies Adam7 interlacing, 'keep' preserves existing interlacing; default is 0):
+
+`oxipng --interlace {{type}} {{path/to/file.png}}`
```
Comment (spageektti) [inline pages/common/oxipng.md:24]:
```suggestion
- Set the PNG interlacing type (`0` removes interlacing, `1` applies Adam7 interlacing, `keep` preserves existing interlacing; default is `0`):

`oxipng --interlace {{0|1|keep}} {{path/to/file.png}}`
```

**Review by spageektti — APPROVED** [review]
Thanks for your contribution

**Pull request merged**